### DISCUSSION TITLE
HDF5 backend for Entropy

### DISCUSSION THREAD

**Discussion by galwiner** [entropy-lab/entropy#29] [Ideas]
Entropy saving should support HDF5 as a backend. 

We need to make sure QPU-DB is on equal footing here.

**Comment by galwiner** [comment]
@talshani ,@qguyk  we need to discuss if the QPU-DB is a resource that should be save on the "persistence backend" or not.

**Comment by galwiner** [comment]
Our understanding is now that the HDF5 format is more suitable as a **single experiment record**, not a database.

What we want to do is to add the ability to export records from the persistence DB (sqlite, or anything else) to HDF5.

HDF5 attrs will contain experiment metadata in a key:value format, allowing users to define what metadata they want to have. 

Entropy will generate HDF5 files automatically for each node. 

Users will be able to export an experiment or many experiments to HDF5 based on a query on the persistence DB.

**Reply by galwiner** [reply]
@urig 
@Gilbert-W

**Comment by talshani** [comment]
see pull-request #51 implementing this backend
